app/backend/src/couchers/errors.py

```diff
@@ -119,6 +119,7 @@
 SIGNUP_FLOW_BASIC_NEEDED = "Basic part needed to sign up."
 SIGNUP_FLOW_EMAIL_STARTED_SIGNUP = "Please check your email for a link to continue signing up."
 SIGNUP_FLOW_EMAIL_TAKEN = "That email address is already associated with an account. Please log in instead!"
+SIGNUP_FLOW_USERNAME_TAKEN="That username is already associated with an account"
```

Comment:
Please follow the coding style and use spaces around `=`.

The `black` auto-formatter will do this for you.